### PR TITLE
PS-5103: Post push fix, fixing a crash with background scrubbing

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

File: storage/innobase/dict/dict0load.cc [hunk 1]
```diff
@@ -3196,95 +3196,105 @@ void dict_load_tablespaces_for_upgrade() {
...
-/***********************************************************************//**
-Loads a table id based on the index id.
-@return true if found */
-static
-bool
-dict_load_table_id_on_index_id(
-/*===========================*/
-        space_index_t           index_id,  /*!< in: index id */
-        table_id_t*             table_id) /*!< out: table id */
+/***********************************************************************/ /**
```
Comment (laurynas-biveinis) [inline storage/innobase/dict/dict0load.cc]: Please new style function header comment (no /*****....*****/, no /*=======*/, function arg comments in the header)

File: storage/innobase/dict/dict0load.cc [hunk 2]
```diff
@@ -3196,95 +3196,105 @@ void dict_load_tablespaces_for_upgrade() {
...
+  return (found);
 }
 
-dict_table_t*
-dict_table_open_on_index_id(
-/*========================*/
-        space_index_t index_id,    /*!< in: index id */
-        bool dict_locked)       /*!< in: dict locked */
+dict_table_t *dict_table_open_on_index_id(
+    /*========================*/
```
Comment (laurynas-biveinis) [inline storage/innobase/dict/dict0load.cc]: Likewise

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: storage/innobase/dict/dict0load.cc (hunk 2 above)
Comment (dutow) [reply storage/innobase/dict/dict0load.cc]: Updated with the comments removed.

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by dutow** [comment]
https://ps.cd.percona.com/job/percona-server-8.0-param/65/

**Pull request merged**